docs/src/main/asciidoc/Installation_Guide.adoc

```diff
@@ -86,16 +86,41 @@ provision customized WildFly server using Galleon.
 Started Developing Applications Guide] shows you how to build Jakarta EE
 applications and deploy them to WildFly.
 
+[[WildFly_Maven_Plugin_Provisioning]]
+= Installing WildFly with the WildFly Maven Plugin
+
+The link:https://docs.wildfly.org/wildfly-maven-plugin[WildFly Maven Plugin] exposes a `package` goal that allows you 
+to provision a customized WildFly installation containing your application.
```

Comment:
Do you want to mention any of the other goals here? You can use `provision` or `image` to provision a server as well. Maybe we don't reference `image` here, but `provision` does seem relevant.

It's possibly better to have two sections. "Installing WildFly with the WildFly Maven Plugin" and "Installing WIldFly with the WildFly Maven Plugin for your Application".

Reply:
I am not sure about a new complete chapter. They share the same Galleon configuration. I just introduced the two other goals.